browser-extension/common/src/storage.ts

```diff
@@ -40,6 +40,14 @@ export const clearAllRecords = async (): Promise<void> => {
   await chrome.storage[STORAGE_TYPE].clear();
 };
 
+export const clearAllRecordsExceptVariables = async (): Promise<void> => {
+  const allRecords = await getSuperObject();
+  const variableKeys = Object.keys(allRecords).filter((key) => key.startsWith("rq_var_"));
+  const variableRecords = Object.fromEntries(variableKeys.map((key) => [key, allRecords[key]]));
+  await chrome.storage[STORAGE_TYPE].clear();
+  await saveObject(variableRecords);
```

Comment:
Can we do something like this (not exactly this)?

```suggestion
  const extensionStatus = getVariable(Variable.IS_EXTENSION_ENABLED);
  await chrome.storage[STORAGE_TYPE].clear();
  setVariable(Variable.IS_EXTENSION_ENABLED, extensionStatus);
```

Reply:
What if in future we also want to preserve some other variable

Reply:
Let's backup only extension enabled for now. In case we require more, we can modify this to be more generalized later.

Reply:
Lets get and set this key only